app/test_main.py

```diff
@@ -1,3 +1,33 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestHumanAge:
+
+    @pytest.mark.parametrize("cat_age, dog_age, human_age", [
+        pytest.param(0,
+                     0,
+                     [0, 0],
+                     id="Function should return [0, 0] if age equal 0"),
+        pytest.param(14,
+                     14,
+                     [0, 0],
+                     id="Function should round floor(down)"),
+        pytest.param(15,
+                     15,
+                     [1, 1],
+                     id="Function should count years"),
```

Comment:
you done well for edged situation - that 14/15 is 0/1, let's do the same for the edged situations.